### PR TITLE
changed the FreeCAD... startup "logo"

### PR DESCRIPTION
**Pull request by Zolko-123**
Changed in ~/src/Main/MainGui.cpp
the "FreeCAD..." logo when launching FreeCAD from the command line, replaced the ### by a figlet output:
> figlet "FreeCAD ..."
and escaped the \ by \\
will propose it for MainCmd.cpp too

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

---

### CODE REVIEW

**Comment by yorikvanhavre** [comment]
Personally I find this less readable than the current one.
If we were to change, I would go for an ascii art, then....

**Comment by luzpaz** [comment]
@Zolko-123 it's customary to post this type of proposed change on the UI/UX subforum first. Please open a thread to discuss such things. Thank you.

**Comment by Zolko-123** [comment]
Ah OK, thank-you.

**Comment by yorikvanhavre** [comment]
@Zolko-123 Indeed such changes will very likely generate a lot of discussion among the community (everything that touches visual appearance of FreeCAD always does), and they should be discussed there first, so we don't take any arbitrary decision here that might upset people... I suggest you post about your proposal in the [UX Design section](https://forum.freecadweb.org/viewforum.php?f=34) of the forum. I will close your two pull requests for now, okay? If the community likes your proposal, we can resurrect them anytime. Thanks!

**Comment by Zolko-123** [comment]
>we don't take any arbitrary decision here that might upset people

on the other hand, it clearly states:

*************************************
© Juergen Riegel, Werner Mayer, Yorik van Havre 2001-2018
   _____               ____    _    ____   
  |  ___| __ ___  ___ / ___|  / \  |  _ \   
  | |_ | '__/ _ \/ _ \ |     / _ \ | | | | 
  |  _|| | |  __/  __/ |___ / ___ \| |_| |  _   _   _  
  |_|  |_|  \___|\___|\____/_/   \_\____/  (_) (_) (_) 

*************************************

So if you, Yorik, don't take your responsibilities as copyright holder, and need to refer to the wider community for trivial changes like this, I don't see why you should have a special treatment and have your name as copyright holder. Which means that I very much understand that you don't like such changes, but I very much don't understand that they need to be discussed in a way where you know in advance that it's going to be unproductive. May I thus propose to remove you from the copyright holders list:

**************************
© Juergen Riegel, Werner Mayer 2001-2018
   _____               ____    _    ____   
  |  ___| __ ___  ___ / ___|  / \  |  _ \   
  | |_ | '__/ _ \/ _ \ |     / _ \ | | | | 
  |  _|| | |  __/  __/ |___ / ___ \| |_| |  _   _   _  
  |_|  |_|  \___|\___|\____/_/   \_\____/  (_) (_) (_) 
**************************

in other words, I bloody fucking don't care about this subject, and certainly won't loose my time in discussing this in the forum. I don't know if you follow the forum, but I spend my time in trying to make an assembly framework for FreeCAD ("assembly without solver") and I'll stick with that.

For your information, I'm from the same generation X as Linus, and the snowflakes attitude and politically correctness is not my thing. 

Regards,

Zoltan

**Comment by yorikvanhavre** [comment]
> May I thus propose to remove you from the copyright holders list:
I actually totally agree with you ;)
